content/docs/start/experiments.md

```diff
@@ -25,6 +25,11 @@ $ dvc run -n evaluate \
 
 ### 💡 Expand to see what happens under the hood.
 
+The `-M` flag here specifies a metrics file, while the `--plots-no-cache` flag
+specifies a plot metrics file produced by this stage that will not tracked by
+DVC. You can choose whether to ignore the files in DVC or cache them. See
+[`dvc run`](/doc/command-reference/run#options) for use cases.
```

Comment:
what do you mean by `use cases`?

Reply:
As in when is it appropriate to cache the files produced by stages into DVC Cache and when is it not. 
![Screenshot from 2020-07-02 02-27-44](https://user-images.githubusercontent.com/32031518/86290811-afe67680-bc0b-11ea-88ff-55a14e127e47.png)
To give a brief idea to users about caching files produced by the stage since we are not writing anything here about `-m` and `--plots` directly.

Reply:
By use cases we mean https://dvc.org/doc/use-cases @utkarshsingh99 I think you just want to say "for more details" (about the options). And in fact it's a good idea to link to `dvc run` somewhere because I just noticed it's not linked anywhere. Maybe just change the sentence "DVC generates a new stage..." to "`dvc run` generates..." (using back ticks, so it gets linked automatically to the command ref. — no need for manual md link).